Code/utility_functions.py

```diff
@@ -210,3 +218,20 @@ def last_minima(self, all_data):
     min_times = time[angle_max_index]
 
     return min_times
+
+def total_angle(be, se0, se1):
+    """
+    Takes encoder values at a single time and converts to a total angle.
+    Used for triple pendulum examples.
+    """
+    x, y = position_seat_cartesian(be, se0, se1)
+    total_angle = np.arctan(x/y)
+    return total_angle
+
+def total_angle_list(all_data):
+    """returns the list of total_encoder values and matching times"""
+    total_angles = []
```

Comment:
I don't get what this does, you are looping over times but not using i or t in anything, so this just does that one line a lot of times? Also what is it used for?